backend/app/assets/javascripts/spree/backend/sticky.coffee

```diff
@@ -0,0 +1,106 @@
+$.fn.offsetBottom = ->
+  $(this).offset().top + $(this).height()
+
+class Viewport
+  measure: =>
+    @top = document.documentElement.scrollTop || document.body.scrollTop
+    @height = $(window).height()
+    @bottom = @top + @height
+
+  isScrollingUp: =>
+    @measure()
+    goingUp = @top < @lastTop
+    @lastTop = @top
+    goingUp
+
+  isAbove: (el) =>
+    @top < el.offset().top
+
+  isBelow: (el) =>
+    @bottom > el.offsetBottom()
+
+  fits: (el) =>
+    el.height() < @height
+
+viewport = new Viewport()
+
+$(window).on
```

Comment:
Could we only bind to global events if we have stickies on the page in the first place?